kurl_proxy/cmd/main.go

```diff
@@ -317,7 +317,7 @@ func getHttpsServer(upstream, dexUpstream *url.URL, tlsSecretName string, secret
 			} else {
 				delete(secret.Annotations, "acceptAnonymousUploads")
 			}
-			_, err = secrets.Update(secret)
+			_, err = secrets.Update(context.Background(), secret, metav1.UpdateOptions{})
```

Comment:
This is after the request context is closed so it must be background

https://github.com/replicatedhq/kots/pull/1749/files#diff-db27cebad99585565557ed130999dc35ca6e4d2e4cf648b04119580339d9d407R297